### ISSUE TITLE
wcag support p-autocomplete

### ISSUE THREAD

**Issue by jaxiell** [primefaces/primeng#4189]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please fork the plunkr below and create a case demonstrating your bug report. Issues without a plunkr have much less possibility to be reviewed.

https://www.primefaces.org/primeng/#/autocomplete

**Current behavior**

p-autocomplete doesn't meet wcag 2.1 level A requitement. However, it mentioned support  in Wiki/Accessibility page with Moderate level support.

**Expected behavior**

p-autocomplete should have accessible label applied. A label associated with a form control ensures that information about the form control is spoken by screen readers when it receives focus.

**Minimal reproduction of the problem with instructions**

1. https://www.primefaces.org/primeng/#/autocomplete
2. check the p-autocomplete input element,it doesn't have label.
3. run WCAG inspector agaist page. and it shows forms controls must have label rule violation.

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.X

* **PrimeNG version:** 2.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =
